src/test/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperatorTest.scala

```diff
@@ -136,6 +136,63 @@ class TezosNodeOperatorTest extends FlatSpec with MockFactory with Matchers with
     val results = pages.toList
     results should have length 1
     results.head.futureValue should have length 1
- }
+  }
+
+  "getLatestBlocks" should "correctly fetch blocks starting from a given head" in {
+    //given
+    val tezosRPCInterface = stub[TezosRPCInterface]
+    val blockResponse = Future.successful(TezosResponseBuilder.blockResponse)
+    val operationsResponse = Future.successful(TezosResponseBuilder.operationsResponse)
+    val votesPeriodKind = Future.successful(TezosResponseBuilder.votesPeriodKind)
+    val votesQuorum = Future.successful(TezosResponseBuilder.votesQuorum)
+    val votesProposal = Future.successful(TezosResponseBuilder.votesProposal)
+
+    (tezosRPCInterface.runAsyncGetQuery _)
+      .when("zeronet", "blocks/BLJKK4VRwZk7qzw64NfErGv69X4iWngdzfBABULks3Nd33grU6c~")
```

Comment:
Would it make sense to use any different block hash for testing this call, as to make it plainly different from the one that refers to the "head" block?

Reply:
I don't get it. Looking at builders this is a "head" block only by convention.

Reply:
What I meant is that looking at these two test cases, it's not clear what they are doing differently. I'm always more worried about the fact that reading the code makes the logic plain to see, than the code works itself.
Of course when doing reviews it strikes out immediately if something is not so evident.

In the end I'm just highlighting the fact that it took me some effort to identify any difference in the test code, as to check if there might've been some error in the logic.